### ISSUE TITLE
Can't select everything if line creation is selected in the toolbar

### ISSUE THREAD

**Issue by a22jacka** [HGustavs/LenaSYS#14679]
## Description
If line-creation is selected in the toolbar then pressing <kbd>ctrl</kbd> + <kbd>A</kbd> doesn't do anything. May happen for other things in the toolbar aswell

## Job
Fix this

**Comment by b22davbe** [comment]
This problem does only occur if the line-creation is selected and not if you're in any other modes. We made it work so that the issue would be solved but realised that it made the program worse. It doesn't really make sense with how the line-creation behaves in this program to be able to mark everything while in the line-creation mode. When we tested it and pressed CTRL + A while in line-creation mode it marked everything but if you then try to click anywhere on the canvas the program tries to make a line between two of the elements resulting in a for the most of the time unwanted line.  

To summarize we recommend to keep this as it is but we have the code for the issue to be solved.

**Comment by a22solmi** [comment]
There should be a key event for ctrl + a. You should try to change the mouse pointer mode to "select" when pressing "ctrl + a".
